best-practices/README.md

```diff
@@ -189,10 +189,9 @@ Email
 
 JavaScript
 ----------
-* Use Coffeescript, ES6 with [babel], or another language that compiles to
-  JavaScript
+* Use JavaScript with [babel], or another language that compiles to JavaScript.
```

Comment:
I prefer not being explicit on versions, since tooling might change throughout the course of time.
What benefits do you think it brings @joshuaclayton ?